src/bindings/js/node/include/core_set_property_args.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <napi.h>
+
+#include "openvino/runtime/core.hpp"
+
+/**
+ * @brief This struct retrieves data from Napi::CallbackInfo.
+ */
+struct CoreSetPropertyArgs {
+    std::string device_name;
+    ov::AnyMap parameters;
+
+    CoreSetPropertyArgs() {}
+    CoreSetPropertyArgs(const Napi::CallbackInfo& info) {
+        const size_t args_length = info.Length();
+
+        if (!is_valid_input(args_length, info))
```

Comment:
I think the exception can be thrown from the `is_valid_input` method.  If-statement with the negation of the condition is less clear and unnecessary.

Reply:
Then, it has call earlier, from original method for example